File: python/2020/03/25/2.py
def isOdd(num):
    if num%2==0:
        return True
    else:
        return False

def main():
    num=int(input("请输入一个数字："))
    if isOdd(num):
        print("这是个偶数")
    else:
        print("这不是个偶数")

main()

File: c/2019/11/09/2.cpp
#include<stdio.h>
int main(){
    int factorial(int);
    while (true){
        printf("Please input a number, this progamme will make a factorial for it.\n");
        printf("Your number i\s:");
        int num;
        scanf("%d",&num);
        if (num<0){
            printf("End of Progamme.\n");
            break;
        }
        printf("%d\'s factorial is %d.\n",num,factorial(num));
    }
}

int factorial(int n){
    int res=1;
    for(int i=1;i<=n;i++){
        res=res*i;
    }
    return res;
}